### DISCUSSION TITLE
Record multiple responses for a stimulus via "audio-keyboard-response"

### DISCUSSION THREAD

**Discussion by xinyiguan** [jspsych/jsPsych#2564] [Q&A]
I wonder how can I record a series of keyboard responses for a single audio stimulus by using the "audio-keyboard-response" plugin. How do I save the data of multiple rt's and responses for one stimulus?

**Comment by jodeleeuw** [comment]
Adding a link in here to the duplicate question: https://github.com/jspsych/jsPsych/discussions/1302#discussioncomment-2484263
